mmv1/third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -887,6 +895,17 @@ func privateNetworkCustomizeDiff(_ context.Context, d *schema.ResourceDiff, meta
 	return nil
 }
 
+// The private path for Google Cloud Services option of the IP configuration is supported only on MySQL and PostgreSQL database types.
+// The API returns an invalid request error for an attempt to enable or explicitly disable this option on SQL Server type instances.
+func privatePathForGoogleCloudServicesSqlServerOnlyCustomizeDiff(_ context.Context, diff *schema.ResourceDiff, v interface{}) error {
```

Comment:
This makes sense, but we typically try to leave validation like this to the server, to reduce complexity in the client and reduce the burden of keeping up with API changes. What happens if we don't have the CustomizeDiff? Specifically, I'm wondering if there is an error message that a user could take action on.

Reply:
The SQL Admin returns a Bad Request 400 error when the `privatePathForGoogleCloudServices` options is set to either true or false for SQLSERVER instances. So this raw error is visible to developers and they can know how to act on it.
Thus I have removed this custom validation in https://github.com/GoogleCloudPlatform/magic-modules/pull/6986/commits/db73c43deb143c458fb5420745cf811e292a42be.